packages/arb-token-bridge-ui/src/components/TransferPanel/TokenInfoTooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { shortenAddress } from '../../util/CommonUtils'
+import { captureSentryErrorWithExtraData } from '../../util/SentryUtils'
+import { ExternalLink } from '../common/ExternalLink'
+import { Tooltip } from '../common/Tooltip'
+import { TokenLogo } from './TokenLogo'
+import { useNativeCurrency } from '../../hooks/useNativeCurrency'
+
+const createBlockExplorerUrlForToken = ({
+  explorerLink,
+  tokenAddress
+}: {
+  explorerLink: string | undefined
+  tokenAddress: string | undefined
+}): string | undefined => {
+  if (!explorerLink) {
+    return undefined
+  }
+  if (!tokenAddress) {
+    return undefined
+  }
+  try {
+    const url = new URL(explorerLink)
+    url.pathname += `token/${tokenAddress}`
+    return url.toString()
+  } catch (error) {
+    captureSentryErrorWithExtraData({
+      error,
+      originFunction: 'createBlockExplorerUrlForToken'
+    })
+    return undefined
+  }
+}
+
+export const TokenInfoTooltip = ({
+  token
+}: {
+  token: ERC20BridgeToken | null
+}) => {
+  const [networks] = useNetworks()
+  const { isDepositMode, childChainProvider } =
+    useNetworksRelationship(networks)
+  const childChainNativeCurrency = useNativeCurrency({
+    provider: childChainProvider
+  })
+
+  if (!token) {
+    return <span>{childChainNativeCurrency.symbol}</span>
+  }
+
+  const tokenAddress = isDepositMode ? token.address : token.l2Address
+
+  const href = createBlockExplorerUrlForToken({
+    explorerLink: networks.sourceChain.blockExplorers?.default.url,
```

Comment:
it should be the destination chain's explorer link

"You will receive on..."

<img width="404" alt="image" src="https://github.com/user-attachments/assets/37ff47ca-dd90-4119-bc38-841728090746" />

Reply:
Updated